test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -0,0 +1,113 @@
+import * as path from 'path';
+import { DAPP_URL, largeDelayMs, openDapp, unlockWallet } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+
+/**
+ * Default options for setting up Multichain E2E test environment
+ */
+export const DEFAULT_OPTIONS = {
```

Comment:
```suggestion
export const DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS = {
```
[nit]

Reply:
resolved in [396ec6f](https://github.com/MetaMask/metamask-extension/pull/29298/commits/396ec6ffcf728182695a52a8f42b9c4b72122933)